apps/rsautl.c

```diff
@@ -98,10 +98,12 @@ OPTIONS rsautl_options[] = {
     {"oaep", OPT_OAEP, '-', "Use PKCS#1 OAEP"},
     {"sign", OPT_SIGN, '-', "Sign with private key"},
     {"verify", OPT_VERIFY, '-', "Verify with public key"},
-    {"asn1parse", OPT_ASN1PARSE, '-'},
+    {"asn1parse", OPT_ASN1PARSE, '-',
+     "asn1parse the output data, useful with -verify"},
```

Comment:
change comma to semicolon.
Maybe "Run output through asn1parse; useful with -verify" as the text?